watertap/unit_models/MD/membrane_distillation_0D.py

```diff
@@ -406,24 +409,30 @@ def _add_mass_transfer(self):
             doc="Mass transfer from feed to permeate",
         )
         def eq_connect_mass_transfer(b, t, p, j):
-            return (
-                b.cold_ch.mass_transfer_term[t, p, j]
-                == -b.hot_ch.mass_transfer_term[t, p, j]
-            )
+
+            if p == "Liq":
+                return (
+                    b.cold_ch.mass_transfer_term[t, "Liq", j]
+                    == -b.hot_ch.mass_transfer_term[t, "Liq", j]
+                )
+
+            else:
+                b.cold_ch.mass_transfer_term[t, p, j].fix(0)
```

Comment:
How is `hot_ch.mass_transfer_term` handled for vapor?

Reply:
Is it essentially just a stale variable?

Reply:
@adam-a-a The hot channel does not include a mass_transfer_term for the vapor phase because it uses solution property packages that exclude vapor phase properties. Instead, the vapor state block is managed independently using a separate vapor property package, and it does not affect the main state blocks of the channel control volume